CHANGELOG.md

```diff
@@ -27,6 +31,7 @@
 ### Fixed
 
 - Corrected the conditional compilation of the seccomp rule for `madvise`.
+- Corrected the seccomp filter when building with glibc.
```

Comment:
I think this is not worth mentioning. I would mention it only if correcting this would result in succesful glibc compilation. If that is the case right now, I would rephrase this to focus on that instead.

Reply:
The commit that was adding the seccomp whitelist for glibc was tested so that Firecracker built with glibc successfully runs a VM. It's not about just building Firecracker.

Reply:
Yes, now Firecracker correctly works when built with glibc (although since we don't have functional tests for glibc, there is no guarantee on how long this will be the case).
Since this was something broken and now works correctly, I think it's worth including in the changelog.

Reply:
One more thing: wasn't the documentation added in the same PR? I noticed that this entry is in 0.15.2 while the documentation for glibc is in 0.16.0.